app/src/main/java/org/fossasia/openevent/general/event/TicketService.kt

```diff
@@ -0,0 +1,10 @@
+package org.fossasia.openevent.general.event
+
+import io.reactivex.Flowable
+
+class TicketService(private val ticketApi: TicketApi, private val ticketsDao: TicketsDao) {
+
+    fun getTickets(id: Long): Flowable<List<Ticket>> {
+        return ticketApi.getTickets(id)
```

Comment:
@iamareebjamal I want to update the logic of saving it in database in a new PR as this one has become quite big

Reply:
OK, but the schema of the ticket should be correct if committed. So either remove that completely or fix it

Reply:
Thanks :+1: